### DISCUSSION TITLE
Scroll controlled volume

### DISCUSSION THREAD

**Discussion by WallK** [cpiber/NebulaEnhance#4] [Ideas]
I'd really like to have a feature where mousewheel scrolling anywhere in the player would control volume.
Preferably with logarithmic scale and percent indication.
This extension does it for the YouTube: https://chrome.google.com/webstore/detail/youtubetm-mouse-controls/odhhfodnkaidhocaaaokdofhkoiahpbn

Thank you!

**Comment by cpiber** [comment]
I'll look into that!

**Reply by cpiber** [reply]
It wasn't too hard, could you please try it out and see if it's how you imagined?

You can download the extension here: 
[enhancer_for_nebula-0.3.1.zip](https://github.com/cpiber/NebulaEnhance/files/7021112/enhancer_for_nebula-0.3.1.zip)

To install (in Chrome), please go to Setting, click Extensions (lower left), enable Developer Mode (upper right) and select "Load unpacked". Here select the unzipped folder (should contain folders _locale, icons, scripts, styles).

**Reply by WallK** [reply]
Yes! Thank you!
Two things I would also suggest:
I think it will be good to add an option to hide volume percentage for people who don't use scroll control
Logarithmic has a very aggressive curve even with the minimal 0.05 change rate

**Reply by cpiber** [reply]
I think I'll tie it to the same control (only show volume if scroll enabled), at least for now.

You can actually put in whatever you want (even negative numbers). I read in an article (https://www.dr-lex.be/info-stuff/volumecontrols.html) that x^4 is a good approximation, I have to agree, x^2 feels better (more control).
